webrisk/webrisk_client_system_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webrisk
+
+import (
+	"context"
+	"flag"
+	"testing"
+	"time"
+
+	pb "github.com/GoogleCloudPlatform/golang-samples/webrisk/internal/webrisk_proto"
+)
+
+// The system tests below are non-deterministic and operate by performing
+// network requests against the Web Risk API servers. Thus, in order to
+// operate they need the user's API key. This can be specified using the -apikey
+// command-line flag when running the tests.
+var apiKeyFlag = flag.String("apikey", "", "specify your Web Risk API key")
```

Comment:
Normally, samples are tested against live services to make sure they really work. If we want to do that here, this should be an environment variable so we don't need custom testing logic.

Reply:
Done.